packages/control-input/src/js/react.js

```diff
@@ -0,0 +1,99 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * control-input function
+ *
+ * Control inputs include radio buttons and checkboxes. They allow users to select one or more options.
+ *
+ **************************************************************************************************************************************************************/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+ * The checkbox component
+ *
+ * @param  {string}   label    - The label
+ * @param  {string}   name     - The name of the input element
+ * @param  {string}   value    - The value of the input element
+ * @param  {string}   id       - The id of the input element, for additional labels
+ * @param  {boolean}  checked  - The check option
+ * @param  {boolean}  disabled - The disable option
+ * @param  {boolean}  full     - The full variant option
+ * @param  {function} onChange - A callback for onChange
+ */
+export const Checkbox = ({ label, name, value, id, checked, disabled, full, onChange }) => {
+	const attributeOptions = {};
+
+	if( typeof id !== 'undefined' ) {
```

Comment:
I tend to agree but since the control inputs are rendered with label that don't need IDs (as it wraps the input field) I think we can ease off here a little. The component by itself is a11y without it's context.